Curtains/Curtains.Domain/Models/BedspreadsModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class BedspreadsModel
+    {
+        public BedspreadsModel()
+        {
+            Sets = new HashSet<SetModel>();
+            UserOrders = new HashSet<UserOrderModel>();
+        }
+        public int Id { get; set; }
+        public string Title { get; set; } = null!;
+        public string Size { get; set; } = null!;
```

Comment:
we missed description here

Reply:
Now there is a description field in BedspreadModel class